index.html

```diff
@@ -0,0 +1,417 @@
+                        <button type="button"> REQUEST AN OFFER <iconify-icon icon="iconoir:nav-arrow-down" style="color: white;" width="20" height="20" rotate="270deg"></iconify-icon> </button>
+                    </div>
+                    <div class="help-bg-color"> 
+                    </div>
+                </div>
+            </div>
+            <div class="column">
+                <img src="./images/help-one.png" alt="">
+            </div>
+        </div>
+    </section>
+
+<!-- tenth section - inspire -->
+
+    <section class="inspire has-text-black has-text-centered">
+        <h2 class="is-size-1-desktop mb-6 pb-4">Allow yourself to be inspired</h2>
+        <div class="columns">
+            <div class="column">
+                <img src="./images/inspire/1_wine.jpg" alt="">
+                <div class="text">
+                    <p class="title">References from the world of wine</p>
+                    <p class="tags">#decoratedbydecoglas #designedbydecoglas #references #wine</p>
+                </div>
+            </div>
+            <div class="column">
+                <img src="./images/inspire/2_Schaumwein.jpg" alt="">
+                <div class="text">
+                    <p class="title">References from the world of sparkling wine</p>
+                    <p class="tags">#decoratedbydecoglas #designedbydecoglas #references #sparklingwine</p>
+                </div>
+            </div>
+            <div class="column">
+                <img src="./images/inspire/3_Alkoholfreie-Getraenke.jpg" alt="">
+                <div class="text">
+                    <p class="title">References from the world of soft drinks</p>
+                    <p class="tags">#decoratedbydecoglas #designedbydecoglas #references #softdrinks</p>
+                </div>
+            </div>
+            <div class="column">
+                <img src="./images/inspire/4_Spirituosen.jpg" alt="">
+                <div class="text">
+                    <p class="title">References from the world of spirits</p>
+                    <p class="tags">#decoratedbydecoglas #designedbydecoglas #references #spirits</p>
+                </div>
+            </div>
+        </div>
+<!-- inspire section second row -->
+        <div class="columns">
+            <div class="column">
+                <img src="./images/inspire/5_Beer.jpg" alt="">
+                <div class="text">
+                    <p class="title">References from the world of beer</p>
+                    <p class="tags">#beer #decoratedbydecoglas #designedbydecoglas #references</p>
+                </div>
+            </div>
+            <div class="column">
+                <img src="./images/inspire/6_honey.jpg" alt="">
+                <div class="text">
+                    <p class="title">References from the world of food</p>
+                    <p class="tags">#decoratedbydecoglas #designedbydecoglas #food #references</p>
```

Comment:
out of a curiosity, why you choose tag here. also space between the text are not aligned with site